Vizzuality/half-earth-v3
# Fix/nrc dropdowns

## Challenges and Ranking filter dropdowns fixes
### Description
Missing `slug`s have been added to the dropdown options to allow the proper behaviour of the optionsSelection functions (updating the challenges filter in the url and updating the sorting order on the country ranking)
### Testing instructions
Go to National Report Cards and select a country. On the country view select the Challenges and Ranking tabs and play with the filtering and sorting dropdowns. The chart and the ranking should respond to the dropdown option selection.
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-180
